src/app/views/query-response/QueryResponse.tsx

```diff
@@ -48,10 +52,29 @@ class QueryResponse extends Component<IQueryResponseProps, IQueryResponseState>
     this.setState({ showShareQueryDialog: !this.state.showShareQueryDialog });
   };
 
-  public toggleModal = () => {
+  public toggleExpandResponse = () => {
     this.setState({ showModal: !this.state.showModal });
   }
 
+  public toggleModal = (event: any) => {
+    const { key } = event;
+    if (key && key.includes('expand')) {
+      this.toggleExpandResponse();
+    }
+    if (key && key.includes('share')) {
+      this.handleShareQuery();
+    }
+  }
+
+  public renderItemLink(link: any) {
+    return (
+      <TooltipHost content={link.title} id={getId()} calloutProps={{ gapSpace: 0 }} >
+        <Icon iconName={link.itemIcon} style={{ paddingRight: 5 }} />
+        {link.headerText}
```

Comment:
Here we should show the translated text instead

Reply:
where? at the `{link.headertext}` ??

Reply:
Nvm I get what you meant